docker/puppetserver-standalone/docker-entrypoint.d/20-use-templates-initially.sh

```diff
@@ -11,3 +11,11 @@ do
     test -f "$TEMPLATES/$f" && cp -np "$TEMPLATES/$f" .
 done
 cd /
+
+if [ -d /var/tmp/puppetserver/vendored-jruby-gems ]; then
```

Comment:
I think we might want to copy the entire `/opt/puppetlabs/server/data/puppetserver` directory as that seems more future proof?

Also seems like the best generic way to correctly handle the VOLUME on LCOW given https://github.com/moby/moby/issues/39892

Thoughts?

Reply:
Addendum - as the code is written, that approach won't work for upgrades.

But maybe we could check if the `/opt/puppetlabs/server/data/puppetserver/` is empty and copy it all over to handle the LCOW scenario?

Reply:
I'll take a look and see if I can come up with something to gracefully handle additional contents in that directory. Right now the only things in `data/puppetserver` other than `vendored-jruby-gems` are empty directories, but we would likely want to create those if they don't exist at runtime to work around the windows issue.

Mostly want to make sure we aren't cleaning up anything other than `vendored-jruby-gems` since that's the only directory there I wouldn't expect to persist on container upgrades (since we'd always want the gems to match what was packaged).

Reply:
Just saw the addendum, might make sense to copy everything over if `data/puppetserver` is empty, and otherwise cleanup and copy over `vendored-jruby-gems` ? That doesn't feel terrible

Reply:
FYI - this is what I tentatively have over in `pe-puppetserver`: https://github.com/puppetlabs/pe-puppetserver/pull/381/commits/bbdbbef3996c95e20af0194ff99485d2f696d2e3

Reply:
I had something very similar